pom.xml

```diff
@@ -36,68 +36,179 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <tez.version>0.9.1</tez.version>
-    <hive.version>2.3.7</hive.version>
+    <hive.version>2.3.9</hive.version>
     <license.maven.plugin.version>3.0</license.maven.plugin.version>
     <hive.execution.engine>mr</hive.execution.engine>
     <junit.jupiter.version>5.7.1</junit.jupiter.version>
     <mockito.version>3.8.0</mockito.version>
+    <log4j2.version>2.17.1</log4j2.version>
```

Comment:
The move to `log4j2` should also be mentioned in the release notes as this could potentially break projects which have configured their tests and HiveRunner with `log4j`.